drake/util/TrigPoly.h

```diff
@@ -5,6 +5,29 @@
 #include <map>
 //#include <iostream>  // just for debugging
```

Comment:
Can we lose this while we're in here?  It doesn't seem referenced from anywhere commented out either...